broker/core/scheduler-jobs/test/jobs.Scheduler.spec.js

```diff
@@ -284,6 +284,7 @@ describe('Jobs', function () {
     });
     after(function () {
       sandbox.restore();
+      logSpy.restore();
```

Comment:
In this UT, `logSpy` which is a sinon spy for logger(error) is not restored. This was creating an issue with other UT in `cf.TokenIssuer.spec.js` as it was trying to wrap the same function. Earlier, it was not creating issue as the order of execution was opposite for these two UTs and the cf UT has already restored the function.